gatsby/src/components/AtoZListPage/AtoZListPage.tsx

```diff
@@ -45,11 +44,19 @@ export const AtoZListPage: FC<AtoZListPageProps> = ({
 
 	const alphabet = () => (
 		<Alphabet>
-			{letters.map(({ letter }) => (
-				<Letter key={letter} to={`#${letter}`}>
-					{letter.trim().toUpperCase() || "n/a"}
-				</Letter>
-			))}
+			{letters.map(({ letter }) => {
+				const trimmedLetter = letter.trim().toUpperCase();
+				return (
+					<Letter key={letter} to={`#${letter}`}>
+						{trimmedLetter ? (
+							<span className="visually-hidden">
+								{`Browse ${title.toLocaleLowerCase()} by the letter `}
```

Comment:
Feel like the title maybe wants title case or to be in quotes? But not sure if this makes a difference when we're just looking at screen readers?

Reply:
screen reader won't care but I will get rid of the lowercasing anyway